lib/ansible/plugins/lookup/sops.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+#
+#  Copyright 2018 Edoardo Tenani <e.tenani@arduino.cc> [@endorama]
+#
+# This file is part of Ansible.
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.errors import AnsibleError, AnsibleParserError, AnsibleLookupError
+from ansible.plugins.lookup import LookupBase
+from ansible.module_utils._text import to_text, to_native
+
+from ansible.utils.display import Display
+display = Display()
+
+from subprocess import Popen, PIPE
+
+DOCUMENTATION = """
+    lookup: sops
+    author: Edoardo Tenani <e.tenani@arduino.cc>
+    version_added: "2.8"
+    short_description: Read sops encrypted file contents
+    description:
+        - This lookup returns the contents from a file on the Ansible controller's file system.
+        - This lookup requires the sops executable to be available in the controller PATH.
+    options:
+        _terms:
+            description: path(s) of files to read
+            required: True
+        print:
```

Comment:
What is this option needed for? After all, these are secrets you're dealing with, and if the user wants to print them she can use the `debug` module.

Reply:
You're perfectly right, actually I didn't thought about that. I'll remove it.

Reply:
As suggested, see a262523